ChangeLog

```diff
@@ -1,3 +1,10 @@
+2015-02-23	Aditi Gupta	<agupta@msu.edu>
+
+   * sandbox/{collect-reads.py, correct-errors.py, 
```

Comment:
This paragraph should be indented by one more column to match the formatting of the rest of the ChangeLog entries.